src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -610,5 +665,55 @@ public SAMSequenceDictionary getBestAvailableSequenceDictionary() {
             return sequenceDictionary;
         }
     }
+
+    /**
+     * This class is a hack to force parallel loading of the headers and indexes of remote gvcf files.
+     * It initializes a feature reader and starts a query.  This causes the header and index to be read, and also causes any
+     * pre-fetching to begin if enabled.  It is very narrowly crafted and should not be used for other purposes.
+     */
+    private static class InitializedQueryWrapper implements FeatureReader<VariantContext> {
+        private final FeatureReader<VariantContext> reader;
+        private final SimpleInterval interval;
+        private final CloseableTribbleIterator<VariantContext> query;
+
+        private InitializedQueryWrapper(final FeatureReader<VariantContext> reader, final Locatable interval) throws IOException {
+            this.reader = reader;
+            this.interval = new SimpleInterval(interval);
+            this.query = reader.query(interval.getContig(), interval.getStart(), interval.getEnd());
+        }
+
+        @Override
+        public CloseableTribbleIterator<VariantContext> query(final String chr, final int start, final int end) throws IOException {
+            final SimpleInterval queryInterval = new SimpleInterval(chr, start, end);
+            if( !interval.equals(queryInterval)){
+                throw new GATKException("Cannot call query with different interval, expected:" + this.interval + " queried with: " + queryInterval );
+            }
+            if( query != null){
+                return query;
+            } else {
+                throw new GATKException("Cannot call query twice on this wrapper.");
```

Comment:
Hmn, I had obviously considered that.  Done now.  Not that it makes very much difference other than to alert us if genomicsdb changes internally in a way that this doesn't make sense any more...